docs/releases/v2020.2.rst

```diff
@@ -0,0 +1,178 @@
+Gluon 2020.2
+=============
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+* GL.iNet
+
+  - GL-AR750S
+
+* TP-Link
+
+  - CPE220 (v3)
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+* EnGenius
+
+  - ENS620EXT [#outdoor]_
+
+* Linksys
+
+  - EA6350 (v3)
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+* Cudy
+
+  - WR1000
+
+lantiq-xway
+~~~~~~~~~~~
+
+* NETGEAR
+
+  - DGN3500B
+
+lantiq-xrx200
+~~~~~~~~~~~~~
+
+* TP-Link
+
+  - TD-W8970
+
+
+x86-legacy [#newtarget]_
+~~~~~~~~~~
+
+* Devices older than the Pentium 4
+
+
+.. [#newtarget]
+  This is a new target in this release
+
+
+.. [#oudoor]
+  This device is supposed to be setup outdoors and will therefore have it's outdoor mode flag automatically enabled.
+
+
+Major changes
+-------------
+
+Device Classes
+==============
+
+Devices not can be categorised into device classes. This device class can determine which features
+as well as packages are installed on the device when building images.
+
+Yurrently there are two classes used in Gluon, tiny and standard. All devices with less than 64M of Memory or
+7M of usable firmware space are assigned to the tiny class.
```

Comment:
Style: repeat the "less than": "less than 64MiB of RAM or less than 7MiB of usable firmware space"